### ISSUE TITLE
Show and hide individual tracks in a bookmark list

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#5503]
Now a checkmark hides or displays all bookmark list's items (bookmarks or tracks). If a list has many tracks (or if we start recording tracks later into one list, e.g. named "Recorded Tracks"), then showing and hiding individual tracks becomes an important feature.

AFAIR `<visibility>` KML tag can allow us to show and hide individual tracks.

**Comment by dandud100** [comment]
I agree to this, it would be good if we can't just hide the complete track list, but also only single tracks inside of it.

**Comment by hoizan** [comment]
Also, for hiding and unhiding individual bookmarks within a list.

**Comment by Aurareus** [comment]
+1
This is functionality that would be extremely useful/preferable to me.
I need to be able to toggle the visibility of individual tracks, as it is right now that means I have to save every track to a separate list which creates a very cluttered and disorganized Bookmarks and Tracks section.
Additionally, since all of my tracks need to be separate lists, I have no option to collectively toggle the visibility of all my tracks at once, so when I need to do that I have to individually go through and toggle the visible of them all one by one.

**Comment by patepelo** [comment]
#Emails&Reviews

Android review, 4 stars: "Probably one of the best apps for OSM. It still has its flaws, but nothing fundamental. For the saved routes view, I miss the option to display only 1 route. I display all of them at once and it's confusing. If it were possible to add this choice."

**Comment by Rob-from-VI** [comment]
Is anyone working on this?   It seems to be a very important issue to make the track recording feature more useable, as we hikers soon build long lists of recorded tracks!!!

It would enable us to save tracks in lists for a region, but only display the desired current track.

**Comment by biodranik** [comment]
Agree. A workaround is to move a track into a separate list and hide the old list.

**Comment by cmacfarl** [comment]
Came here specifically looking for this exact feature on the Android build.

What might be the timeline for merging https://github.com/organicmaps/organicmaps/pull/12021?  It seems from the conversation that it is largely complete, but the original developer doesn't have the bandwidth at present to push it over the line?

**Comment by Rob-from-VI** [comment]
One of the betas had a working prototype, just a wee bit inelegant.  We shall hope, and those who know how might pray..., or help

**Comment by 8z4kg7f452-hub** [comment]
Are there any plans to bring this to iOS and Mac?
At the moment managing tracks is a cumbersome task.

**Comment by biodranik** [comment]
Yes, in progress.
